### ISSUE TITLE
Incorporate Aaron Swartz data into RECAP

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#668]
This is split off from #448, which suggests that the Aaron Swartz data (or at least part of it) can be found here: 

https://archive.org/search.php?query=%22pacer%20dump%22

I started digging into this data a bit today, and have it filed away in my bulk data folder. It looks like a tricky one to incorporate. This screenshot describes it well:

![screenshot from 2017-04-19 11-16-28](https://cloud.githubusercontent.com/assets/236970/25195188/aaa95944-24f1-11e7-8536-b83cdb512216.png)

Basically, it's a zip file containing numerical directories, each of which contains PDFs, a docket.html file and a summary.html file that has even more metadata. 

This data seems to represent an older version of PACER, so incorporating it will mean making some custom parsers. It's nothing terribly complicated, but it'll need custom work.

**Comment by mlissner** [comment]
Doing some quick math on this to get a feel for size:

 - There are about 217GB of data here.
 - The AKD data has 54435 pdfs and is 8.0GB in size.
 - Extrapolating this, there's about 1.5M PDFs in this collection.

Not bad at all.

**Comment by mlissner** [comment]
Looks like what's on IA is only a fraction of the collection. The rest is [maybe held by PRO (see table at bottom of page)][pro] or possibly by somebody else that can look into it if PRO can't dig it up.

[pro]: https://public.resource.org/uscourts.gov/index.html

**Comment by mlissner** [comment]
I'm pretty sure we'll never do this, and we've probably imported most of it already, assuming it wasn't already in the archive when we took it over (which I think it was?)
